fe/fe-core/src/main/java/org/apache/doris/catalog/Database.java

```diff
@@ -605,6 +628,7 @@ public void readFields(DataInput in) throws IOException {
         clusterName = Text.readString(in);
         dbState = DbState.valueOf(Text.readString(in));
         attachDbName = Text.readString(in);
+        dbComment = Text.readString(in);
```

Comment:
u should add a new mete version, other wise, use new version FE read old meta data will fail